README.md

```diff
@@ -48,10 +48,17 @@ The `master` branch tracks the ongoing efforts towards the next stable release t
 
 ## Quick Install
 
+**Linux / Mac**
+
 By using the installer to install the latest stable version:
 ```bash
 bash <(curl -s https://install.aeternity.io/install.sh)
 ```
+See the documentation for [starting](https://docs.aeternity.io/en/stable/operation/#start-the-node) and [configuring](https://docs.aeternity.io/en/stable/configuration/#example) the node.
+
+
+
+**Linux / Mac / Win**
```

Comment:
Win?

Reply:
oof good catch @velzevur , let me find someone to test on windows...